config/environments/development.rb

```diff
@@ -1,6 +1,10 @@
+require 'repository_container'
+require 'item_marketplace/active_record_repository/active_record_item_repository'
+
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  RepositoryContainer.add(:item, ActiveRecordRepository::ItemRepository.new)
```

Comment:
I would prefer something like `RepositoryLoader.load_repositories` here and `RepositoryLoader.load_test_repositories` in the test env file. You shouldn't have update your env files every time you add/remove a repo or update a key name.